src/screens/Signup/styles.module.scss

```diff
@@ -1,4 +1,7 @@
-@import 'variables/colors';
+@import 'variables/_colors';
+@import 'variables/animations';
+
+:export { cerulean: $cerulean; }
```

Comment:
Didn't know this, maybe it would be useful to move it to Spinner styles file? Because you will have to do the same in every component that uses the Spinner and this color seems to be the primary/default one